decidim-meetings/spec/commands/leave_meeting_spec.rb

```diff
@@ -31,6 +31,16 @@ module Decidim::Meetings
       end
     end
 
+    context "when user answered a questionaire" do
```

Comment:
I think this is a better naming:

```suggestion
    context "when user didn't answer the questionaire" do
```

Reply:
The name has been changed to "when the questionnaire/registration form is missing", as i am explicitly testing a scenario when the form is missing.